src/apps/advanced-search-v2/components/AdvancedSearchForm.tsx

```diff
@@ -0,0 +1,190 @@
+import { SEARCH_TERM_OPTIONS } from "../lib/search-fields-config";
+import { INITIAL_PRE_SEARCH_FACETS_STATE } from "../lib/initial-state";
+import MultiSelect from "./MultiSelect";
+import AdvancedSearchAgeSelect from "./AdvancedSearchAgeSelect";
+import AdvancedSearchPublicationYearSelect from "./AdvancedSearchPublicationYearSelect";
+import { Button } from "../../../components/Buttons/Button";
+
+const AdvancedSearchForm: React.FC = () => {
+  const t = useText();
+  const [focusIndex, setFocusIndex] = useState<number | null>(null);
+
+  const {
+    suggests,
+    preSearchFacets,
+    updateSuggest,
+    updatePreSearchFacet,
+    addSuggest,
+    removeSuggest,
+    handleSearch,
+    handleClearFilters
+  } = useSearchFormState();
+
+  const { setView } = useFormVisibility();
+
+  const handleAddSuggest = () => {
+    setFocusIndex(suggests.length);
+    addSuggest();
+  };
+
+  const handleRemoveSuggest = (index: number) => {
+    setFocusIndex(
+      suggests.length > 1 ? Math.min(index, suggests.length - 2) : null
+    );
+    removeSuggest(index);
+  };
+
+  // Check if there are any filters to reset
+  const hasFilters =
+    suggests.some((suggest) => suggest.query.trim()) ||
+    preSearchFacets.some(
+      (preSearchFacet) => preSearchFacet.selectedValues.length > 0
+    );
+
+  const handleSearchComplete = () => {
+    handleSearch();
+    setView("results");
+  };
+
+  return (
+    <section className="advanced-search-v2__form">
+      {/* Suggest inputs */}
+      <div className="advanced-search-v2__suggests">
+        {suggests.map((suggest, index) => {
+          const config =
+            SEARCH_TERM_OPTIONS.find((item) => item.value === suggest.term) ??
+            SEARCH_TERM_OPTIONS[0];
+
+          return (
+            <AdvancedSearchSuggest
+              key={`suggest-${index}`}
+              shouldAutoFocus={index === focusIndex}
+              selectedTerm={suggest.term}
+              onSelectedTermChange={(value) =>
+                updateSuggest(index, { term: value })
+              }
+              query={suggest.query}
+              onQueryChange={(query) => updateSuggest(index, { query })}
+              suggestType={config.type}
+              inputPlaceholder={t(config.placeholderKey)}
+              operator={suggests[index + 1]?.operator}
+              onOperatorChange={
+                index < suggests.length - 1
+                  ? (operator) => updateSuggest(index + 1, { operator })
+                  : undefined
+              }
+              onRemove={() => handleRemoveSuggest(index)}
+              showRemoveButton={suggests.length > 1}
+            />
+          );
+        })}
+
+        <button
+          type="button"
+          className="advanced-search-v2__add-suggest"
+          onClick={handleAddSuggest}
+        >
+          <img src={PlusButtonIcon} alt="" />
+          <span>{t("advancedSearchAddRowText")}</span>
+        </button>
```

Comment:
From an accessiblity perspektive these elements and the related button should probably be updated to a list which is controlled by the button.